packages/react-centra-checkout/src/Context/index.tsx

```diff
@@ -636,21 +636,54 @@ export function useCentraReceipt(token: string): Centra.CheckoutApi.OrderComplet
 /** Returns the latest orders for the currently logged in user
   @param from - Display orders from this index. Defaults to 0.
   @param size - Display this many orders. Defaults lists all orders.
+  @param token.tokenName - The name of the cookie used to use/store as Centra checkout token
+  @param token.tokenExpires -  When the cookie used to store the Centra checkout token will expire, days as a number or a Date
+  @param token.tokenCookieOptions - Cookie options
 */
-export function useCentraOrders(from?: number, size?: number): Centra.CheckoutApi.OrdersResponse {
+export function useCentraOrders(
+  from?: number,
+  size?: number,
+  token?: {
+    tokenName?: string
+    tokenExpires?: number | Date
+    tokenCookieOptions?: Cookies.CookieAttributes
+  },
```

Comment:
**question:** Is this necessary?
**suggestion:** Would be nice if we can enforce that `tokenName` in both the `Context` as well as inside `useCentraOrders`.